geode-redis/src/distributedTest/java/org/apache/geode/redis/executors/general/ExistsDUnitTest.java

```diff
@@ -12,7 +12,7 @@
  * or implied. See the License for the specific language governing permissions and limitations under
  * the License.
  */
-package org.apache.geode.redis.executors;
+package org.apache.geode.redis.executors.general;
```

Comment:
can we rename this package to keys instead of general, as per the dropdown on https://redis.io/commands

Reply:
Makes perfect sense. Done!